content/en/docs/tasks/administer-cluster/kubeadm/change-package-repository.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Changing the Kubernetes package repository
+min-kubernetes-server-version: 1.24
+content_type: task
+weight: 120
+---
+
+<!-- overview -->
+
+This page explains how to switch from one to another Kubernetes package repository
+upon upgrading from one to another Kubernetes minor release. Unlike Google-hosted
+repositories, the Kubernetes community-owned package repositories are structured
+in a way that there's a dedicated package repository for each Kubernetes minor version.
+
+
+For more information about the Kubernetes community-owned package repositories,
+see the ["pkgs.k8s.io: Introducing Kubernetes community-owned package repositories"](TBD)
+blog post.
+
+## {{% heading "prerequisites" %}}
+
+This document assumes that you're already using the Kubernetes community-owned
+package repositories. If that's not the case, it's strongly recommend to migrate
+from the Google-hosted repository to the Kubernetes package repositories
+as described in [the blog post](TBD).
+
+If you're unsure if you're using the Kubernetes package repositories or the
+Google-hosted repository, the first step of this document explains how to check
+what repository is in use.
+
+<!-- steps -->
+
+## Verifying if the Kubernetes package repositories are used
+
+The purpose of this step is to verify if you're using the Kubernetes package repositories.
+
+{{< tabs name="k8s_install_versions" >}}
+{{% tab name="Ubuntu, Debian or HypriotOS" %}}
+
+Print contents of the file that defines the Kubernetes `apt` repository:
+
+```shell
+cat /etc/apt/sources.list.d/kubernetes.list
+```
+
+If you see the URL similar to:
+
+```
+deb https://pkgs.k8s.io/core:/stable:/v{{< skew currentVersionAddMinor -1 "." >}}/deb/ /
+```
+
+**You're using the Kubernetes package repositories and this guide applies to you.**
+Otherwise, it's strongly recommend to migrate to the Kubernetes package repositories
+as described in [the blog post](TBD).
```

Comment:
FIXME

Reply:
Well aware of it, I still didn't receive date for blog post which is part of the URL, but as soon as I do, I'll update this. :)